### DISCUSSION TITLE
why get bad result?

### DISCUSSION THREAD

**Discussion by greasebig** [blepping/comfyui_jankhidiffusion#24] [Q&A]
![Uploading image.png…]()
![Uploading image.png…]()

however in diffusers, it works well. same inference config.
sdxl

**Comment by greasebig** [comment]
<img width="1081" alt="image" src="https://github.com/blepping/comfyui_jankhidiffusion/assets/121388156/38c359ec-2b0c-421f-8c6e-efc7a870014a">

**Comment by greasebig** [comment]
did you implement original code completely?

**Comment by blepping** [comment]
> did you implement original code completely?

like the README says: "My implementation may or may not work correctly."

i tried to understand the code and i made a best effort attempt to implement it correctly, but there's definitely no guarantee that it's perfect. if anyone finds a specific part that i didn't implement correctly and points it out, i'd naturally try to fix it.

i use it with SD 1.5 and it works well. i haven't seen very good results with SDXL, i'm not sure what the difference is if it works fine in the diffusers implementation.
